src/fields/equation_output.cc

```diff
@@ -21,14 +21,23 @@ namespace IT = Input::Type;
 
 IT::Record &EquationOutput::get_input_type() {
 
+    static const IT::Selection &discrete_sel =
+        IT::Selection("Discrete_output", "Discrete type of output. Determines type of output data (element, node, native etc).")
```

Comment:
Generalize to 'InterpolationType' selection.
- 'node_data' -> 'P1_average'
- 'elem_data' -> 'P0_value'
- 'corner_data' -> 'D1_value'
- 'native_data' -> 'Native'